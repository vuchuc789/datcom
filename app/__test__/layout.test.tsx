import { render } from '@testing-library/react';
import RootLayout from '../layout';

describe('Root Layout', () => {
  test('render correctly', () => {
    const spiedConsoleError = jest.spyOn(console, 'error');

    // mock console.error because validateDOMNesting function throws an error when rendering
    // eslint-disable-next-line @typescript-eslint/no-empty-function
    spiedConsoleError.mockImplementation(() => {});

    const { container } = render(<RootLayout>sample text</RootLayout>);

    expect(container).toMatchSnapshot();

    spiedConsoleError.mockRestore();
  });
});

export {};
