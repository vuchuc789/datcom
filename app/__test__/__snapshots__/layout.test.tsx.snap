// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Layout render correctly 1`] = `
<div>
  <html
    lang="en"
  >
    <head />
    <body>
      sample text
    </body>
  </html>
</div>
`;
